hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -824,6 +824,17 @@ def alter(table_name_str, wait = true, *args)
               tdb.removeValue(name)
             end
             hasTableUpdate = true
+          elsif method == 'table_remove_coprocessor'
+            classname = arg.delete(CLASSNAME)
+            raise(ArgumentError, 'CLASSNAME parameter missing for table_remove_coprocessor method') unless classname
```

Comment:
nit: we allow it to be an array, should we say "CLASSNAMES"?

Reply:
I would prefer to keep using classname without plural form, basically, I'm following the convention of this file and other place if they're defined as array are using single form.